dist/bin/kctf-cluster

```diff
@@ -105,11 +125,37 @@ function kctf_cluster_start_gce {
 
   # Cloud DNS
 
-  if [ ! -z "${DOMAIN_NAME}" ]
-  then
-    DNS_ZONE=$(gcloud dns managed-zones list --filter "dns_name:${DOMAIN_NAME}" --format 'get(name)')
+  if [ ! -z "${DOMAIN_NAME}" ]; then
+    ZONE_NAME="$(echo ${DOMAIN_NAME} | sed 's/[.]/--/g')-dns-zone"
```

Comment:
if you change the zone from foo.com to bar.com you have a name conflict which means you would have to delete the foo.com zone first. This is undesirable for two reasons:
1) deleting the zone would allow a third party to take it over (IIUC)
2) deleting it is hard, you would need to list all records first and delete them before you can delete the zone